packages/docs/site/docusaurus.config.js

```diff
@@ -146,6 +157,10 @@ const config = {
 						className: 'header-github-link',
 						'aria-label': 'GitHub repository',
 					},
+					// {
```

Comment:
Can we remove this?

Reply:
I see now this is for the language switcher.

When I uncomment this, the switcher doesn't work.
Let's make sure we have a way for users to switch languages.